Godeps/_workspace/src/k8s.io/kubernetes/pkg/kubectl/resource_printer.go

```diff
@@ -333,17 +335,19 @@ type PrintOptions struct {
 // received from watches.
 type HumanReadablePrinter struct {
 	handlerMap map[reflect.Type]*handlerEntry
-	options    PrintOptions
+	Options    PrintOptions
```

Comment:
See how `NewHumanReadablePrinter` takes all other values as arguments. Perhaps changing the function signature to take the other new options explicitly would be better.
@juanvallejo wouldn't that work?